core/src/test/java/org/jdbi/v3/core/transaction/TestSerializableTransactionRunner.java

```diff
@@ -38,38 +40,46 @@ public void setUp() throws Exception
     {
         db = Jdbi.create(dbRule.getConnectionFactory());
         db.setTransactionHandler(new SerializableTransactionRunner());
+        db.configure(SerializableTransactionRunner.Configuration.class, config -> config.setMaxRetries(RETRIES));
```

Comment:
Tip: `configure()` is intended to allow chaining method calls on the callee (db in this instance).

When you just want to set config and don't need to chain on the callee, `getConfig(Class)` is a little simpler to use:

```java
db.getConfig(SerializableTransactionRunner.Configuration.class).setMaxRetries(RETRIES);
```